spec/models/lti_deployment_spec.rb

```diff
@@ -143,4 +143,167 @@
       end
     end
   end
+  describe '#create_or_update_lti_assessment' do
+    let(:course) { create :course }
+    let(:lti_deployment) { create :lti_deployment, course: course }
+    let(:assessment) { create :assignment, course: course }
+    let(:scope) { 'https://purl.imsglobal.org/spec/lti-ags/scope/lineitem' }
+    let(:lti_service_lineitem) { create :lti_service_lineitem, lti_deployment: lti_deployment }
+    before :each do
+      stub_request(:post, Settings.lti.token_endpoint)
```

Comment:
What happened to the idea of creating a `send_request` method and just stubbing that?

Reply:
Please do that instead of this. This is going to be unnecessarily difficult to maintain.

Reply:
Per discussion this will be a separate PR